frontend/src/components/MapView/Legends/ColorIndicator/index.tsx

```diff
@@ -12,22 +18,34 @@ const useStyles = makeStyles(() =>
       height: 10,
       width: 10,
       marginRight: 4,
-      backgroundColor: ({
-        color,
-        opacity,
-      }: {
-        color: string;
-        opacity: number;
-      }) => fade(color, opacity),
+      backgroundColor: ({ color, opacity }: StylesProps) =>
+        fade(color, opacity),
+    },
+    fillPatternIndicator: {
+      height: 10,
+      width: 10,
+      marginRight: 4,
```

Comment:
since these are shared maybe this could be extracted into a common indicator class?
and the `fillPatternIndicator` overrides just what's necessary?

Reply:
Done